### PR TITLE
Add extended_description to line item

### PR DESCRIPTION
**Pull request by jdrieghe**
### Added
- `extended_description` to line item on `/invoices.info`
- `extended_description` to line item on `/creditNotes.info`
- `extended_description` to line item on `/quotations.info`

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1893,6 +1893,7 @@ Get details for a single invoice.
                                 + type: `productCategory` (string)
                             + quantity: 3 (number)
                             + description: `An awesome product` (string)
+                            + extended_description: `Some more information about this awesome product` (string)
```
Comment (WouterSioen) [inline apiary.apib]: Q: is this an html field in the application? Otherwise it might be useful to note that it has to be send as markdown?

Comment (jdrieghe) [reply apiary.apib]: I don't think so, but I'll verify.

Comment (jdrieghe) [reply apiary.apib]: Hmm, apparently it is. Should we add that to the documentation here? Do we do that in any other places?

Comment (WouterSioen) [reply apiary.apib]: I would add it as a comment. Not sure if this is already documented for other fields or if there is a datatype for it. I'll check.

Comment (WouterSioen) [reply apiary.apib]: So, apparently it is not documented anywhere 🤦‍♂️

Comment (WouterSioen) [reply apiary.apib]: I'll make sure this gets documented for the core endpoints.

Comment (WouterSioen) [reply apiary.apib]: Maybe not that important for the read side though, but should especially be specified for write endpoints imo.

 :ok: by @WouterSioen on 12/06/2018 10:25:04

**Review by WouterSioen — APPROVED** [review]

**Review by WouterSioen — DISMISSED** [review]
Need to build again

**Review by WouterSioen — DISMISSED** [review]

**Review by luisaterrabrito — DISMISSED** [review]

**Comment by WouterSioen** [comment]
As Dietrich so kindly pointed out to me in my PR, Markdown is written with a capital `M`.

**Review by jenssegers — DISMISSED** [review]

**Comment by jenssegers** [comment]
Also what @WouterSioen said!

**Review by WouterSioen — APPROVED** [review]

**Review by steftrenson — APPROVED** [review]

**Review by adognameddog — APPROVED** [review]

**Pull request merged**